modules/products/jira/locals.tf

```diff
@@ -49,4 +51,6 @@ locals {
       additionalJvmArgs = ["-Dcom.atlassian.jira.status.index.check=false"]
     }
   }) : yamlencode({})
+
+  license = "AAAB8Q0ODAoPeNp9Uk1z2jAQvetXaKY3ZmSMSRrKjA+JraZkgu2xTTv9Ogh7AVEjeSSZ1P++wpi2CSQHHbQfb9++t+++QIlva4VHY+yOp9eTqTfB9/Mce67nobUCEBtZ16CcR16A0EBLbrgUPo1ymibpLKMoanZLUPFqoUFpf+SiQArDChOxHfjbLUixRkmjig3TEDID/gGbuFdkNEY9at7W0JWH9DN9jBOanjL0d81V27UlI8/9dAKnc8arE3oGag9qFvp39IaSq/CbR97HDzfkfvxhgrZcMadWsmwK4xw+RMuVeWIKHIvD9+Ab1QB6sJmkr7LjWADCvCjJmqUuFK87AbrIBYEurNIxeF2/wWAQxTn5GKckSeNwEeSzOCKLjNqEHyiwu5d42WKzAdyjYCoKWYLCdq8tFAZ/3xhT/5gOh2vpMFMxrTkTTiF3w+rYQeDY8dPBocRCGlxybRRfNgYsMtfYSFw02sidddFBVl5hQDBRnFtgeQUpvc1pSO6+Hkj2Nryh9NmN9ItYzxbil5BPAmU08u0j1657RDq5AOpc7Mww9TfxxtxndbFaM8E16wyYtziQu5qJFnUa29jL4wzhn905aIN7LfFKWuWrZs0FLmEPlbSU9DN+/59NR+8scJnv60dE96xqjtRXrNKA/gDQakzsMCwCFG8JD+9JKcQy6O+SAw69it4/5qBuAhQCz6xRgHMTic87Kqty52VlAGeQxw==X02nj"
```

Comment:
Why do we put license in code? This is a sensitive data and should never expose publicly.

Reply:
My bad, removed